### ISSUE TITLE
All Trial jobs failed from MNIST example of nni

### ISSUE THREAD

**Issue by yiqiaoc11** [microsoft/nni#5325]
**Describe the issue**:
All trail failed on "nnictl create --config nni-2.10\examples\trials\mnist-pytorch\config_detailed.yml".
But I can run it properly without nni
(base) C:\Users\vanil>python nni-2.10\examples\trials\mnist-pytorch\mnist.py
C:\Users\vanil\anaconda3\lib\site-packages\nni\runtime\platform\standalone.py:32: RuntimeWarning: Running trial code without runtime. Please check the tutorial if you are new to NNI: https://nni.readthedocs.io/en/stable/tutorials/hpo_quickstart_pytorch/main.html
  warnings.warn(warning_message, RuntimeWarning)
{'data_dir': './data', 'batch_size': 64, 'batch_num': None, 'hidden_size': 512, 'lr': 0.01, 'momentum': 0.5, 'epochs': 10, 'seed': 1, 'no_cuda': False, 'log_interval': 1000}
[2023-01-29 17:30:13] ←[32mIntermediate result: 97.01  (Index 0)←[0m
[2023-01-29 17:31:03] ←[32mIntermediate result: 98.1  (Index 1)←[0m
[2023-01-29 17:31:56] ←[32mIntermediate result: 98.42  (Index 2)←[0m
[2023-01-29 17:33:06] ←[32mIntermediate result: 98.56  (Index 3)←[0m

**Environment**:
- NNI version: 2.10
- Training service (local|remote|pai|aml|etc): local
- Client OS: Windows 10
- Server OS (for remote mode only):
- Python version: 3.8
- PyTorch/TensorFlow version: 1.10
- Is conda/virtualenv/venv used?: Yes
- Is running in Docker?: No

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:
{
  "params": {
    "experimentType": "hpo",
    "trialCommand": "python mnist.py",
    "trialCodeDirectory": "C:\\Users\\vanil\\nni-2.10\\examples\\trials\\mnist-pytorch",
    "trialConcurrency": 4,
    "trialGpuNumber": 0,
    "maxExperimentDuration": "1h",
    "maxTrialNumber": 10,
    "useAnnotation": false,
    "debug": false,
    "logLevel": "info",
    "experimentWorkingDirectory": "C:\\Users\\vanil\\nni-experiments",
    "tuner": {
      "name": "TPE",
      "classArgs": {
        "optimize_mode": "maximize"
      }
    },
    "trainingService": {
      "platform": "local",
      "trialCommand": "python mnist.py",
      "trialCodeDirectory": "C:\\Users\\vanil\\nni-2.10\\examples\\trials\\mnist-pytorch",
      "trialGpuNumber": 0,
      "debug": false,
      "useActiveGpu": true,
      "maxTrialNumberPerGpu": 1,
      "reuseMode": false
    }
  },
  "execDuration": "42s",
  "nextSequenceId": 12,
  "revision": 17
}

**Log message**:
 - nnimanager.log:

[2023-01-29 17:51:20] INFO (main) Start NNI manager
[2023-01-29 17:51:20] INFO (NNIDataStore) Datastore initialization done
[2023-01-29 17:51:20] INFO (RestServer) Starting REST server at port 8080, URL prefix: "/"
[2023-01-29 17:51:20] INFO (RestServer) REST server started.
[2023-01-29 17:51:20] INFO (NNIManager) Starting experiment: ae130u9f
[2023-01-29 17:51:20] INFO (NNIManager) Setup training service...
[2023-01-29 17:51:20] INFO (LocalTrainingService) Construct local machine training service.
[2023-01-29 17:51:20] INFO (NNIManager) Setup tuner...
[2023-01-29 17:51:20] INFO (NNIManager) Change NNIManager status from: INITIALIZED to: RUNNING
[2023-01-29 17:51:22] INFO (NNIManager) Add event listeners
[2023-01-29 17:51:22] INFO (LocalTrainingService) Run local machine training service.
[2023-01-29 17:51:22] INFO (NNIManager) NNIManager received command from dispatcher: ID, 
[2023-01-29 17:51:22] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 0, "parameter_source": "algorithm", "parameters": {"batch_size": 16, "hidden_size": 128, "lr": 0.001, "momentum": 0.8719471722758799}, "parameter_index": 0}
[2023-01-29 17:51:22] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 1, "parameter_source": "algorithm", "parameters": {"batch_size": 64, "hidden_size": 1024, "lr": 0.1, "momentum": 0.9431249121895658}, "parameter_index": 0}
[2023-01-29 17:51:22] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 2, "parameter_source": "algorithm", "parameters": {"batch_size": 16, "hidden_size": 512, "lr": 0.0001, "momentum": 0.4171615260590785}, "parameter_index": 0}
[2023-01-29 17:51:22] INFO (NNIManager) NNIManager received command from dispatcher: TR, {"parameter_id": 3, "parameter_source": "algorithm", "parameters": {"batch_size": 32, "hidden_size": 512, "lr": 0.0001, "momentum": 0.3612376708356295}, "parameter_index": 0}
[2023-01-29 17:51:27] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 0,
  hyperParameters: {
    value: '{"parameter_id": 0, "parameter_source": "algorithm", "parameters": {"batch_size": 16, "hidden_size": 128, "lr": 0.001, "momentum": 0.8719471722758799}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2023-01-29 17:51:27] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 1,
  hyperParameters: {
    value: '{"parameter_id": 1, "parameter_source": "algorithm", "parameters": {"batch_size": 64, "hidden_size": 1024, "lr": 0.1, "momentum": 0.9431249121895658}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2023-01-29 17:51:27] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 2,
  hyperParameters: {
    value: '{"parameter_id": 2, "parameter_source": "algorithm", "parameters": {"batch_size": 16, "hidden_size": 512, "lr": 0.0001, "momentum": 0.4171615260590785}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2023-01-29 17:51:27] INFO (NNIManager) submitTrialJob: form: {
  sequenceId: 3,
  hyperParameters: {
    value: '{"parameter_id": 3, "parameter_source": "algorithm", "parameters": {"batch_size": 32, "hidden_size": 512, "lr": 0.0001, "momentum": 0.3612376708356295}, "parameter_index": 0}',
    index: 0
  },
  placementConstraint: { type: 'None', gpus: [] }
}
[2023-01-29 17:51:37] INFO (NNIManager) Trial job WzycO status changed from WAITING to FAILED
[2023-01-29 17:51:37] INFO (NNIManager) Trial job BaFxU status changed from WAITING to FAILED
[2023-01-29 17:51:37] INFO (NNIManager) Trial job NW7SR status changed from WAITING to FAILED
[2023-01-29 17:51:38] INFO (NNIManager) Trial job ASmzF status changed from WAITING to FAILED

 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**:

**Comment by yiqiaoc11** [comment]
@ liuzhe-lz, Could you please look into this issue which is very fundamental?

**Comment by liuzhe-lz** [comment]
Please check the stderr of failed trials.
The log files are saved at `nni-experiments/trials/<TRIAL-ID>/stderr`. They can also be viewed in web portal.

**Comment by yiqiaoc11** [comment]
fixed.
